### PR TITLE
Update  Object Classification docs for 1.3.2

### LINKED ISSUE #4: object classification: vague description of neighborhood features
The following paragraph needs to be made clearer. In particular, how do the three neighborhood spin boxes relate to the distance transform? And how many features are added for a single ticked checkbox?

You will also notice features, which can be computed “in the neighborhood”. In that case, the neighborhood of the object ( specified by the user at the bottom of the dialog) is found by distance transform and the feature is computed for the object itself and for the neighborhood including and excluding the object.

### LINKED ISSUE #5: object classification: missing explanation of halo size

### PR DESCRIPTION
**Pull request by k-dominik**
fixes https://github.com/ilastik/ilastik.github.io/issues/28

Summary:
* Object Classification
  * [x] updated screenshots
  * [x] document table export

opted for not documenting the object features in this PR.

### CODE REVIEW

**Comment by k-dominik** [comment]
this is ready to be reviewed :)

File: documentation/carving/carving.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ weight: 4
 
 ## Carving Demo (4 minutes)
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/xGyTriPQXHI" frameborder="0" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/xGyTriPQXHI?cc_load_policy=1" frameborder="0" allowfullscreen></iframe>
```
Comment (emilmelnikov) [inline documentation/carving/carving.md]: Could also add `rel=0` query param to show related videos only from the ilastik YouTube channel.

Comment (k-dominik) [reply documentation/carving/carving.md]: cool, I'll update all yt players!

File: documentation/objects/objects.md [hunk 2]
```diff
@@ -53,31 +62,68 @@ If you already have binary segmentation images, skip this section.
 Suppose we have a probability map for a 2-class classification, which looks like this:
 <a href="figs/pixel_results.png" data-toggle="lightbox"><img src="figs/pixel_results.png" class="img-responsive" /></a>
 
-There are two ways to transform a probability map into a segmentation in ilastik and both are covered by the thresholding applet. To see the results of changing the parameter settings in this applet, press the "Apply" button.
+The basic idea of thresholding is to answer a question for _every pixel_ in an image:
+is the pixel value larger than the _threshold value_ or not?
+In ilastik probability maps (result of [Pixel Classification][Pixel Classification Workflow]) pixel values are in the continuous range from `0.0` to `1.0` in each channel and describe the probability that a pixel belongs into a class.
+In this applet this continuous range is transferred into a binary one, containing only `0`s and `1`s (so no values in between) by comparing it to a set threshold.
+
+**Note:** To see any the results of changing the parameter settings in this applet, press the "Apply" button.
```
Comment (akreshuk) [inline documentation/objects/objects.md]:
```suggestion
**Note:** To see the results of changing the parameter settings in this applet, press the "Apply" button.
```

File: documentation/objects/objects.md [hunk 3]
```diff
@@ -53,31 +62,68 @@ If you already have binary segmentation images, skip this section.
...
+
+There are two algorithms you can choose from to threshold your data: _Simple_ and _Hysteresis_, which can be selected using the "Method" drop down.
+
+Both methods share the following parameters:
+ * _Input_ Channel(s): Select on which channel of the probability map to work on.
```
Comment (akreshuk) [inline documentation/objects/objects.md]:
```suggestion
 * _Input_ Channel(s): Select which channel of the probability map contains the objects
```

File: documentation/objects/objects.md [hunk 4]
```diff
@@ -53,31 +62,68 @@ If you already have binary segmentation images, skip this section.
...
+ * _Smooth_: Configure sigmas for smoothing the probability map in order to reduce noise.
+   The Gaussian can be anisotropic, i.e. sigmas for all dimensions can be different.
```
Comment (akreshuk) [inline documentation/objects/objects.md]:
```suggestion
   The Gaussian can be anisotropic, i.e. sigmas for all dimensions can be different. This setting is important for very anisotropic data where we recommend not to smooth across the low-resolution axis at all.
```

File: documentation/objects/objects.md [hunk 5]
```diff
@@ -53,31 +62,68 @@ If you already have binary segmentation images, skip this section.
...
+As for the single threshold case, we provide a view on the intermediate results after the application of the high threshold, the size filter and the low threshold.
+The image below shows the results of the high (detection) threshold after size filter in pink (<span style="color:#ff0aff">&#9679;</span>) overlayed on the final results of the low (segmentation) threshold multiple colors:
 
 <a href="figs/thresholding_two_thresholds.png" data-toggle="lightbox"><img src="figs/thresholding_two_thresholds.png" class="img-responsive" /></a>
 
-The last parameter of this applet is the size filter, for which you can specify the minimum and maximum value. For both thresholding methods the end result is shown in the "Final output" layer.
+An additional setting is available with Hysteresis Thresholding:
+Two (or more) seeds (determined by the "core" threshold) might result in the same object when the segmentation is relaxed.
+With the _Don't merge objects_ checkbox, you can control this behavior:
+Checking it, will preserve one object per detection.
```
Comment (akreshuk) [inline documentation/objects/objects.md]: How about pointing to the separated objects in this image which would get merged if the checkbox is not activate? The light pink and yellow one?

**Pull request merged**